packages/libs/lambda-at-edge/src/routing/redirector.ts

```diff
@@ -0,0 +1,109 @@
+import { compileDestination, matchPath } from "./matcher";
+import { RedirectData, RoutesManifest } from "../../types";
+import * as http from "http";
+
```

Comment:
Now that we have Rollup, we can make the component more maintainable by using different files - it will all get bundled into the handler JS files. Not sure what's a good name, let me know if you have thoughts on this.

Reply:
I think `routing/` makes sense. On a related note, would be nice if at some point we refactored all routing concerns into a `router`, similar to what the [next-server](https://github.com/vercel/next.js/blob/canary/packages/next/next-server/server/router.ts) impl does